src/App.tsx

```diff
@@ -19,17 +21,51 @@ function App() {
     getCurrencies()
   }, [])
 
+  useEffect(() => {
+    const setConvertedValue = async () => {
+      const response = await getConvertedValue(leftCurrency, rightCurrency, leftInputValue)
+      setRightInputValue(response)
+    }
+    setConvertedValue()
+  }, [leftInputValue])
+
+  useEffect(() => {
+    const setConvertedValue = async () => {
+      const response = await getConvertedValue(rightCurrency, leftCurrency, rightInputValue)
+      setLeftInputValue(response)
+    }
+    setConvertedValue()
+  }, [rightInputValue])
+
   return (
     <div className="App">
       <h1>Change money</h1>
 
-      <ListCurrencies allCurrencies={allCurrencies} />
+      <ListCurrencies allCurrencies={allCurrencies} 
+        setLeftCurrency={setLeftCurrency} 
+        setRightCurrency={setRightCurrency}
+        leftCurrency={leftCurrency}
+        rightCurrency={rightCurrency}
+        />
       <div>
-          <input type="text" onClick={() => setIsReversed(prev => !prev)} onChange={(e) => setLeftInputValue(e.target.value)}/>
-          <input type="text" onClick={() => setIsReversed(prev => !prev)} onChange={(e) => setRightInputValue(e.target.value)}/>
+          <input 
+              type="text" 
+              onClick={() => setIsReversed(prev => !prev)} 
```

Comment:
The same question: What happens if I will click only for left input?

Reply:
fixed it
Befor I fixed it in another branch but forgot fixed it here